GVFS/GVFS.Installer/GVFS.Installer.Mac.csproj

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\GVFS.Build\GVFS.cs.props" />
+  <PropertyGroup>
+    <AssemblyName>GVFS.Installer.Mac</AssemblyName>
+    <RootNamespace>GVFS.Installer.Mac</RootNamespace>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <Platforms>x64</Platforms>
+    <RunTimeIdentifiers>osx-x64</RunTimeIdentifiers>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  </PropertyGroup>
+    
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <Version>$(GVFSVersion)</Version>
+    <NoWarn>2008</NoWarn>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <Version>$(GVFSVersion)</Version>
+    <NoWarn>2008</NoWarn>
+  </PropertyGroup>
+  
+  <Target Name="DeleteInstaller" AfterTargets="Clean">
+    <Message Text="Deleting $(TargetDir).." Importance="high"></Message>
+    <RemoveDir Directories="$(TargetDir).." />
+  </Target>
+  
+  <Target Name="AfterCopyFiles" AfterTargets="CopyFilesToOutputDirectory">
+        <Exec Command="/bin/rm -Rf $(TargetDir)*" />

```

Comment:
What is the purpose of this line? Why are we deleting files here?

And nit: fix the spacing

Reply:
```GVFS.Installer.Mac``` project build produces output assemblies (even though no source code file is included in the project.) I could not reliably override the ```csproj``` ```Compile``` or ```Build``` targets to prevent this generation of un-wanted assembly. The work around is to delete the generated assemblies after the are copied to ```TargetDir```. This is not optimal. Will continue to look for options to prevent the generation of unwanted assembly output.

Reply:
removed the spacing